src/components/lisItem.js

```diff
@@ -32,7 +32,7 @@ export const ListItem = ({ name, tel, site, mail, note, newEntry }) => {
 						</span>
 					)}
 					{site && (
-						<a href={`${site}`}>
+						<a href={`${site}`} target="_blank">
```

Comment:
That's unfair, it's forcing a behaviour to the user. 
If they want to open it in a new tab, the will do it themselves.